### DISCUSSION TITLE
Can't edit event filters in Elastic Defend integration -- says permissions are required even though I'm superuser

### DISCUSSION THREAD

**Discussion by iliketurtlesyep** [Security-Onion-Solutions/securityonion#11312] [2.4] [answered]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

16

### RAM

32

### Storage for /

256

### Storage for /nsm

200G

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

I'm trying to add some filters in the Elastic Defend integration but when I go to Artifacts > Event filters > Manage event filters, it presents a Privileges required screen even though I'm logged into an account that has superuser privs.

<img width="487" alt="image" src="https://github.com/Security-Onion-Solutions/securityonion/assets/144930164/681b40b4-ee99-49aa-8062-7cd39faf4948">

<img width="520" alt="image" src="https://github.com/Security-Onion-Solutions/securityonion/assets/144930164/293f8330-7037-44ab-97e5-6663cebea826">

What do I need to do to be able to edit the event filters?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by defensivedepth** [comment, marked as answer]
Go to Kibana --> Stack Management --> Spaces --> Default --> Features  and enable the `Security` features. Then go back and try again.

Also, we are looking at shipping some default event filters.

**Comment by iliketurtlesyep** [comment]
Thank you, that worked perfectly. 

Do you know what filters you're looking to include so we can add them to existing deployments?

**Reply by defensivedepth** [reply]
Still determining that. Unfortunately, the filtering system is not as flexible as Sysmons, so trying to figure out what will work the best.
